rosbag2_test_common/include/rosbag2_test_common/publisher_manager.hpp

```diff
@@ -39,6 +39,40 @@ class PublisherManager
     publisher_nodes_.clear();
   }
 
+  /**
+   * Spin up a Node and publish message to topic_name message_count times at publish_rate.
+   * The Node's scope is tied to the scope of this method.
+   * The message may publish less than message_count times if rclcpp encountered an error.
+   *
+   * @tparam T the type of message to send.
+   * @param topic_name is the name of the topic to publish to.
+   * @param message is the message to publish.
+   * @param publish_rate is the rate to publish the message
+   * @param message_count is the number of times to publish the message.
```

Comment:
Nit: Use `\param` for consistency.

Reply:
woops, CLion defaults to `@param` for me